src/EnergyPlus/OutputReportPredefined.cc

```diff
@@ -1322,6 +1322,11 @@ namespace OutputReportPredefined {
             state.dataOutRptPredefined->tableEntry(state.dataOutRptPredefined->numTableEntry).charEntry =
                 format("{:#12.{}F}", tableEntryReal, sigDigitCount);
         } else {
+            // Formatting in scientific notation, zero sigDigits makes zero sense.
+            // **for something greater than 1E+08**, one sigDigits is very unhelpful (you're having an accuracy of 0.5E+07 at best)
+            if (sigDigitCount < 2) {
+                sigDigitCount = 2;
+            }
```

Comment:
In PreDefTableEntry, if the value we passed is > 1e+08, and we requested an original precision below 2 digits, we reset to 2 digits.

Reply:
This the root cause of #8586. The author of the code figured that 0 precision was good enough because the value was large enough. eg if `12345.6789` is passed, then `12345.` is issued (`{:#12.0F}` is used). But if you suddenly pass a very large value, then it automatically switches to scientific notation, in which case the sigDigits=0 is senseless